ResearchKit/Common/ORKSkin.h

```diff
@@ -187,6 +187,7 @@ typedef NS_ENUM(NSInteger, ORKScreenMetric) {
 
 typedef NS_ENUM(NSInteger, ORKScreenType) {
     ORKScreenTypeiPhoneX,
+    ORKScreenTypeiPhoneMax,
```

Comment:
```suggestion
    ORKScreenTypeiPhoneXSMax,
```

Reply:
Super-duper minor nit - but can we have the name spelled out a iPhoneXSMax, just to stay consistent with the naming format for all other device screen type names?